client/packages/system/src/Patient/DetailView/AddButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import {
+  useTranslation,
+  SplitButton,
+  SplitButtonOption,
+  PlusCircleIcon,
+} from '@openmsupply-client/common';
+// import { usePatient } from '../api';
+import { usePatientModalStore } from '../hooks';
+import { PatientModal } from '.';
+
+export const AddButton = () => {
+  const t = useTranslation('patients');
+  const isDisabled = false; // TODO: usePatient.utils.isDisabled();
```

Comment:
what does `isDisabled` mean? when would a patient be disabled?

Reply:
Only case I can think of is when we are viewing patients from another store and we do not have permission to edit. I'll take this out.. I don't think we need it